paltax/InputConfigs/input_config.py

```diff
@@ -103,13 +103,23 @@ def get_config():
         'exposure_time': 1024, 'num_exposures': 2.0, 'sky_brightness': 22,
         'magnitude_zero_point': 25, 'read_noise': 3.0
     }
-    cosmos_path = str(pathlib.Path(__file__).parent.parent.parent)
-    cosmos_path += '/datasets/cosmos/cosmos_galaxies_train.npz'
+
+    root_path = str(pathlib.Path(__file__).parent.parent.parent)
+    cosmos_path = root_path + '/datasets/weighted_catalog.h5'
+    
+    hdf5_file = h5py.File(cosmos_path, 'r')
+    # These are the weights associated with a cerain parameter
+    # Options are ASYMMETRY, Axial_Ratio, CONCPETRO, GINI, IDENT, M20, RHALFreal, RPETROreal
+    parameter = 'GINI'
+    catalog_weights = hdf5_file['parameters_and_weights'][parameter+'_weights'][:]
```

Comment:
I think the [:] can be replaced with a .value. Also the formatting is a bit odd here. Why are the images saved in galaxy_images directly whereas the parameters and weights first need to be accessed through 'parameters_and_weights'. I think it would be simpler to:

1. Have all the images, parameters, and weights be keys of the hdf5_file (i.e. directly access gini_weights or gini)
2. Unless there's a very specific reason, I would prefer that all the keys be lowercase. For example, `gini` and `gini_weights` instead of `GINI` as you have it now. Listing all of the options in the CONFIG is a nice touch though.

Reply:
It made the most sense to me to group all of the parameters and weights into a common "parameters_and_weights" group, rather than having 16 keys right from the get-go, one for each parameter/weight. But yes, I can change it, these should be easy fixes